toolkit/tools/specreader/specreader.go

```diff
@@ -200,8 +244,44 @@ func createChroot(workerTar, buildDir, specsDir, srpmsDir string) (chroot *safec
 	return
 }
 
+func findSpecFiles(specsDir string, specListSet map[string]bool) (specFiles []string, err error) {
+	// Find the filepath for each spec in the SPECS directory.
+	if len(specListSet) == 0 {
+		var specSearch string
+		specSearch, err = filepath.Abs(filepath.Join(specsDir, "**/*.spec"))
+		if err != nil {
+			err = fmt.Errorf("invalid spec path: '%s'. Error: %w", specsDir, err)
+			return nil, err
+		}
+		specFiles, err = filepath.Glob(specSearch)
+		if err != nil {
+			err = fmt.Errorf("failed to find *.spec files. Check that %s is the correct directory. Error: %w", specsDir, err)
+			return nil, err
+		}
+		return
+	} else {
+		for specName := range specListSet {
+			var matchingSpecFiles []string
+
+			specSearch := filepath.Join(specsDir, fmt.Sprintf("**/%s.spec", specName))
+			matchingSpecFiles, err = filepath.Glob(specSearch)
+
+			// If a SPEC is in the parse list, it should be parsed.
+			if err != nil {
+				return nil, err
```

Comment:
Should we log an error here as well to indicate which spec name we've had issues with or is glob's error already informative enough?

Reply:
I'd expect this to be vanishingly rare, and will likely have nothing to do with which spec we are working on. I've tried a bunch of weird names and haven't managed to break this so far. You would need to actually break the glob search itself, and given linux will take almost anything imaginable as input...

That said, I'll add the search string we are trying to use to the error.